pkg/util/common.go

```diff
@@ -60,3 +63,12 @@ func Keys[K comparable, V any](m map[K]V) []K {
 	}
 	return keys
 }
+
+func PrintTimeCost(fuc string) func() {
+	t := time.Now()
+	return func() {
+		cost := time.Since(t)
+		klog.V(2).Infof("[PrintTimeCost] %s cost %+v", fuc, cost)
+		fmt.Printf("[PrintTimeCost] %s cost %+v", fuc, cost)
```

Comment:
Why do we still need to output this?

Reply:
haha，for I not sure where the output message will be in e2e log.

Reply:
log should be able to output :)